.ccache/ccache.conf

```diff
@@ -0,0 +1,32 @@
+# For complete documentation, see https://ccache.dev/manual/4.6.1.html#_configuration
+
+# This option specifies the maximum size of the cache. Use 0 for no limit. The default value is 5G.
+# Available suffixes: k, M, G, T (decimal) and Ki, Mi, Gi, Ti (binary). The default suffix is G
+max_size = 10G
+
+# This option specifies where ccache will keep its cached compiler outputs.
+cache_dir = /mnt/ccache/
```

Comment:
shouldn't this be in the build directory or under bft's root?

Reply:
No, as his is done on purpose.
First, we need to remember that /mnt/ccache is a folder mount from the host (for containerized).
Also, the default ccache recommended cache is under ${HOME}, see manual.
All the idea of ccache is to boost your compilations time between projects and between builds after 'make clean.
If you put it in the build you are losing most of the idea of the cache. The cache will be deleted when you clean, or when you make a new project or workspace it won't help anything + it will be duplicated in different projects. And I want to share it between projects.

For example, I have 3 vmewathena where I work on vmwathenaX/concord/submodule/concord-bft/, and I also have 3 concord-bft separate workspaces/projects (total 6 projects). When I clean them, I don't want to clean the cache (which is what will happen when calling 'make clean') and I want to share a single cache among 6 projects + I don't want a 'make clean' to clean my cache. So, the idea is that the cache is not part of the build folder. To delete the cache you call make 'clean-ccache-cache'